types/src/front/acl.ts

```diff
@@ -1,29 +1,37 @@
 import { ModelId } from "../shared/model_id";
+import { RoleType } from "./user";
 
 // Supported permissions
-export const SUPPORTED_PERMISSIONS = ["read", "write"] as const;
+export const SUPPORTED_PERMISSIONS = ["admin", "read", "write"] as const;
 
 export type Permission = (typeof SUPPORTED_PERMISSIONS)[number];
 
-// Access Control Entry
-export type ACEType = {
-  groupId: ModelId;
+export type GroupPermission = {
```

Comment:
Sorry for being picky about comments but it's crucial all this code is easy to undertstand.

Can we add a comment here along the lines of:

A GroupPermission is a set of permissions required on a given group.

Even me right now is confused. Is it a required permission or a granted permissions?